src/components/Claim/Link/Initial.view.tsx

```
@@ -355,84 +333,97 @@ export const InitialClaimLinkView = ({
+                        areEvmAddressesEqual(route.toToken, toToken || selectedTokenAddress)
+                )
+
+                if (existingRoute) {
+                    setSelectedRoute(existingRoute)
+                    return existingRoute
+                } else if (!isXChain && !toToken && !toChain) {
+                    setHasFetchedRoute(false)
+                    return undefined
+                }
+
+                const tokenAmount: BigInt = parseUnits(claimLinkData.tokenAmount, claimLinkData.tokenDecimals)
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)